src/Chapter1.hs

```diff
@@ -70,52 +64,53 @@ Each Haskell module starts with the "module <MODULE_NAME> where" line.
 Modules should have the same name as the corresponding file with
 the `.hs` extension.
 -}
-module Chapter1 where
 
-{- |
-In Haskell, we have __expressions__. Expressions can be represented by some
-primitive values (numbers: 1, 100; characters: 'a', 'z'; booleans: True, False;
-etc.) or by a combination of the primitive values and other expressions using
-language syntax constructions (if-then-else, let-in, case-of, etc.) and various
-functions (addition — (+), division — div, maximum — max, sorting — sort,
-sortBy, sortOn, etc.) and variables. Functions are also expressions as well as
-variables.
-
-If an expression is a combination of other values and expressions, it can be
-__evaluated__ (or reduced) to a primitive value. The evaluation process is not
-immediate, since Haskell is a __lazy language__ and it won't evaluate
-expressions unless really necessary. You can see the evaluation results either
-by running a Haskell program or by playing with some functions in the
-interactive interpreter (explained later).
-
-Haskell is a __strongly-typed__ language, which means that each expression has
-a type. Each value and function is associated with some type. You can't change
-the value type. You can only pass a value to some function that will do its
-work, and maybe produce a value of a different type.
-
-Types can be _specific_ (like `Int`, `Integer`, `Double` or `Bool`) and they
-always start with an uppercase letter, or _polymorphic_ (aka general) specified
-through the variables – begin with the lowercase letter. The concept of
-polymorphism is more sophisticated than working with concrete types, thus we
-won't dive too much into it in this chapter and will work with the concrete
-types for now.
-
-Furthermore, Haskell is a __statically-typed__ language, which means that each
-expression has the type known at compile-time, rather than run-time. It allows
-the compiler to catch some kinds of bugs in your program early; before you
-even run it.
-
-Additionally to static typing, Haskell has __type inference__. This means that
-you _don't need_ to specify the type of each expression as it is going to be
-found out for each expression and subexpression by the powerful compiler.
-
-However, you are __strongly encouraged to write top-level function type
-signatures__ and provide types in different situations where you don't
-immediately see what types will be inferred.
--}
+-- | This tutorial uses block comments to explain various concepts and provide
+-- task description.
```

Comment:
Would it be possible for you to configure your autoformatter not to change the style of comments?
We still can review, but it's easier for us when the diff is only for solutions and not for the whole text.

Reply:
Sorry, it was a mistake. Thank you for your patience @chshersh